src/screens/FilterTransactionsScreen.tsx

```diff
@@ -162,66 +171,88 @@ export const FilterTransactionsScreen = (props: Props) => {
+              onPress={() => this.props.updateTransactionStatus(this.isStatusAtive(status) ? '' : status)}
               key={index}
               style={[
                 styles.statusContainer,
-                { borderBottomColor: transactionStatus === status ? palette.secondary : palette.grey },
+                {
+                  borderBottomColor: this.props.filters.transactionStatus === status ? palette.secondary : palette.grey,
+                },
               ]}
             >
-              <Text style={styles.filterText}>{returnStatusCopy(status)}</Text>
+              <Text style={styles.filterText}>{this.returnStatusCopy(status)}</Text>
             </TouchableOpacity>
           ))}
         </ScrollView>
       </View>
       <View style={styles.spacing20}>
-        <InputItem label={label} value={address} editable={false} onChangeText={setAddress} />
+        <InputItem label={label} value={this.props.filters.address} editable={false} onChangeText={this.setAddress} />
         <Image style={styles.image} source={images.nextBlackArrow} />
-        <TouchableOpacity onPress={() => navigateToChooseContactList(label)} style={styles.buttonOverlay} />
+        <TouchableOpacity onPress={() => this.navigateToChooseContactList(label)} style={styles.buttonOverlay} />
       </View>
-      {renderCommonCardContent()}
+      {this.renderCommonCardContent()}
     </View>
   );
-  return (
-    <ScreenTemplate
-      footer={
-        <>
-          <Button title={i18n.filterTransactions.filter} onPress={onFilterButtonPress} />
-        </>
-      }
-      header={<Header navigation={props.navigation} isBackArrow={true} title={i18n.filterTransactions.header} />}
-    >
-      <Calendar isVisible={isCalendarVisible} onDateSelect={onDateSelect} onClose={closeCalendar} />
-
-      <CardGroup
-        onCardPressAction={title =>
-          setTransactionType(title === i18n.filterTransactions.sent ? CONST.send : CONST.receive)
+  render() {
+    return (
+      <ScreenTemplate
+        footer={
+          <>
+            <Button title={i18n.filterTransactions.filter} onPress={this.onFilterButtonPress} />
+          </>
         }
-        label={i18n.filterTransactions.transactionType}
-        cards={[
-          {
-            title: i18n.filterTransactions.received,
-            content: renderCardContent(i18n.filterTransactions.from),
-          },
-          {
-            title: i18n.filterTransactions.sent,
-            content: renderCardContent(i18n.filterTransactions.to),
-          },
-        ]}
-      />
-    </ScreenTemplate>
-  );
+        header={<Header navigation={this.props.navigation} isBackArrow={true} title={i18n.filterTransactions.header} />}
+      >
+        <Calendar
+          isVisible={this.state.isCalendarVisible}
+          onDateSelect={this.onDateSelect}
+          onClose={this.closeCalendar}
+        />
+
+        <CardGroup
+          onCardPressAction={title => this.props.updateTransactionType(title)}
```

Comment:
you have to detect here the translation string and assign CONST.receive and CONST.send accordingly, otherwise, it won't work if the translation will be different than CONST.receive/send value
![Zrzut ekranu 2020-08-28 o 14 37 36](https://user-images.githubusercontent.com/67273035/91561503-09cd9900-e93c-11ea-8633-bafc343f3f5c.png)